RELEASE-NOTES.txt

```diff
@@ -1,11 +1,12 @@
 Unreleased
 ---
+* [**] Tapping on all nested blocks gets focus directly instead of having to tap multiple times depeding on the nesting levels. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/5781]
```

Comment:
There is a typo in the current release note. 
```suggestion
* [**] Tapping on all nested blocks gets focus directly instead of having to tap multiple times depending on the nesting levels. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/5781]
```